core/jazz-ui/src/app/pages/service-metrics/service-metrics.component.html

```diff
@@ -1,138 +1,45 @@
+        <filter-modal #filters [fields]="formFields" (formChange)="applyFilter($event)"></filter-modal>
+      </div>
+    </div>
+    <div [ngSwitch]="sectionStatus" class="jz-flex-expand" style="z-index: 0">
+      <div *ngSwitchCase="'loading'" class="jz-height-full">
+        <div class="jz-absolute-container">
+          <div class="jz-spinner"></div>
         </div>
-        <!-- <advanced-filters #adv_filters (onFilterSelect)='onFilterSelect($event)' [service]='service' [advanced_filter_input]='advanced_filter_input'></advanced-filters> -->
-    <!-- TODO: Remove disclaimer after techtraining -->
-    <div class="disclaimer-static-data hide"><i>This data is for representational purpose only and is not accurate</i></div>
-    
-    <filter-tags #filtertags (OnCancel)='cancelFilter($event)'></filter-tags>
-    
-    <div class="graph-section-container">
-        <div *ngIf=" isGraphLoading || isDataNotAvailable || isError" class='display-widget'>
-            <div class="loading-row-metrics" [class.show]="isGraphLoading">
-                <div class="loading-circle"></div>
-            </div>
-            <div class="loading-row-metrics" [class.show]=" !isGraphLoading && isDataNotAvailable && !isError">
-                <!-- <div class="error-text">Data not available</div> -->
-                <div class="empty-row nodata with-refresh relative">
-                    <div class="icon-icon-nodata"></div>
-                    <div><b>Oops, No Data Found</b><br>Doesn't look like there is any data available here.</div>
-                </div>
-            </div>
-            <div class="error-blk" [class.show]="isError">
-                <div class="empty-row with-refresh">
-                    <div class="icon-icon-server-error">
-                        <span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span><span class="path9"></span><span class="path10"></span>
-                    </div>
-                    <div class='oops'>Oops!</div>
-                    <div>{{errMessage}}<br><a (click)="refreshCostData($event)">REFRESH</a> to give it another try.</div>
-                    <br><div class='ctc-us'>If the problem persists, <a (click)='reportIssue()'> contact us</a>.</div>
-                </div>
-            </div>
+      </div>
+      <div *ngSwitchCase="'error'" class="jz-height-full">
+        <report-issue [response]="errorData.response" [request]="errorData.request" (onRefresh)="refresh()"></report-issue>
+      </div>
+      <div *ngSwitchCase="'empty'" class="jz-height-full jz-flex-centered">
+        <div class="jz-text-center">
+          <div class="jz-section-icon">
+            <span class="icon-icon-nodata"></span>
+          </div>
+          <div><b>Oops, No Data Found</b><br>Doesn't look like there is any data available here.</div>
```

Comment:
`<div><b>Oops, no data found</b><br>The data you requested is currently not available.</div>`

Reply:
Looks like this is not a "new" message but is something we use in a lot of places. So either fix it as a global replace or keep this now and we will create a different issue to update this message.

Reply:
I agree, but I think factoring out the error message is outside of the scope of this story.